Sources/Hyperspace/HTTP/HTTP.swift

```diff
@@ -100,31 +108,52 @@ public struct HTTP {
     public struct Body: Equatable {
         
         /// The raw body data to be attached to the HTTP request
-        public let data: Data
+        public let data: Data?
+        public let additionalHeaders: [HeaderKey: HeaderValue]
         
         /// Initializes a new `HTTP.Body` instance given the raw `Data` to be attached.
-        /// - Parameter data: The raw `Data` to set as the HTTP body.
-        public init(_ data: Data) {
+        /// - Parameters:
+        ///   - data: The raw `Data` to set as the HTTP body.
+        ///   - additionalHeaders: Any additional HTTP headers that should be sent with the request.
+        public init(_ data: Data?, additionalHeaders: [HeaderKey: HeaderValue] = [:]) {
             self.data = data
+            self.additionalHeaders = additionalHeaders
         }
         
-        /// Initializes a new `HTTP.Body` instance given an encodable object.
+        /// Returns a new `HTTP.Body` instance given an encodable object.
         /// - Parameters:
         ///   - encodable: The `Encodable` object to be included in the request.
         ///   - encoder: The `JSONEncoder` to be used to encode the object.
-        public init<E: Encodable>(_ encodable: E, encoder: JSONEncoder = JSONEncoder()) throws {
+        ///   - additionalHeaders: Any additional HTTP headers that should be sent with the request.
+        /// - Returns: Returns a new instance of `HTTP.Body` with the given encodable representation.
```

Comment:
I think the second `Returns` can be removed.
```
/// - Returns: A new instance of `HTTP.Body` with the given encodable representation.
```

Reply:
Fixed